### ISSUE TITLE
Create UserProfile Dataset

### ISSUE THREAD

**Issue by merforga** [GW2Raidar/gw2raidar#89]
As part of #32, a new data set for user profile will need to be created and maintained / updated. 

User Profile should be readily expandable for future requests and information. As a goal, user profile is used by users to review their general performance in raids against global statistics. Some current metrics are:

- Participation levels
- Max and average DPS at ARCHETYPE and ENCOUNTER level
- Down / Death percentages at ARCHETYPE and ENCOUNTER level
- Statistics should be grouped up by GW2 Raidar account > Archetype > Character > Encounter (character level is a nice to have but hopefully won't be too much effort)
- Outgoing boons (dependent on #19
